src/daft-core/src/array/from.rs

```diff
@@ -49,6 +49,21 @@ impl From<(&str, Box<arrow2::array::FixedSizeBinaryArray>)> for FixedSizeBinaryA
     }
 }
 
+impl From<(&str, &[u8])> for FixedSizeBinaryArray {
```

Comment:
would be better to have this take in a `Vec<u8>` rather than a slice that is then copied into a `vec`

Reply:
Updated!